cli/objects/jira_base.py

```diff
@@ -141,6 +143,13 @@ def create_issue(
             self.assign_issue(user_email=assignee, issue=issue.key)
             self.logger.info(f"Issue {issue} has been assigned to user {assignee}")
 
+        if close_issue:
+            self.connection.transition_issue(
+                issue=issue.key,
+                transition="closed",
+                comment="closed by firewatch",
```

Comment:
Maybe link to the repository in the comment as well?

Reply:
appears in description